components/webext-storage/src/store.rs

```diff
@@ -58,10 +58,10 @@ impl Store {
 
     /// Sets one or more JSON key-value pairs for an extension ID. Returns a
     /// list of changes, with existing and new values for each key in `val`.
-    pub fn set(&self, ext_id: &str, val: JsonValue) -> Result<StorageChanges> {
```

Comment:
This is looking great! Sorry I didn't get very far on this, but if you change the UDL to, eg, `StorageChanges set([ByRef] string ext_id, JsonValue val);` these can all remain `&str`. I'll look in more detail next week!